docs/experiments-distilbert_tasb.md

```diff
@@ -0,0 +1,50 @@
+# Pyserini: Reproducing DistilBERT KD TASB Results
+
+This guide provides instructions to reproduce the DistilBERT KD TASB dense retrieval model on the MS MARCO passage ranking task, described in the following paper:
+
+> Sebastian Hofstätter, Sheng-Chieh Lin, Jheng-Hong Yang, Jimmy Lin, Allan Hanbury. [Efficiently Teaching an Effective Dense Retriever with Balanced Topic Aware Sampling
+.](https://arxiv.org/abs/2104.06967) SIGIR2021. 
+
+Starting with v0.12.0, you can reproduce these results directly from the [Pyserini PyPI package](https://pypi.org/project/pyserini/).
```

Comment:
"Starting with v0.12.0"... not true, rewrite...

Reply:
I don't understand what it means. The current version is v0.12.0 and I use this version to run the experiment.

Reply:
Yes, but the pre-built index isn't part of v0.12.0, right? So we'll need to publish a new artifact before the pip version will work fully?

Reply:
As in, you need a dev installation, right?

Reply:
ah, understood. So I remove this part temporarily.

Reply:
yea, we can update to v0.13.0 or something when next package gets published...